starter-code/index.html

```diff
@@ -3,40 +3,67 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+<!--Header-->
+<!--<img id = 'spotify1' src= "images/spotify-logo.png">-->
+
+<div class = "navbar">
+ <ul> Premium </ul>
+ <ul> Discover </ul>
+ <ul> Help </ul>
+ <ul> Download </ul>
+</div>
+
+<!--landing-->
+<div class = "landing">
+
+<h2> Music for everyone. </h2>
+
+<h3 id="subtitle1"> Spotify is now free on mobile, tablet and computer. </h3>
+<h3 id="subtitle2"> Listen to the right music, wherever you are. </h3>
+
+
+<!--features-->
+<div class = "features">
+
+ <h2> What’s on Spotify? </h2> 
+
+  <img id = "songs" src = "music-icon.png">
+  <h3> Millions of Songs </h3>
+   <p>There are millions of songs on Spotify </p>
+
+  <img id = "high-quality" src = "high-quality-icon.png">
+  <h3> HD Music </h3>
+   <p class> Listen to music as if you were listening live</p>
+
+  <img id = "devices" src = "devices-icon.png">
+  <h3> Stream Everywhere</h3>
+   <p> Stream music on your smartphone, tablet or computer</p> 
```

Comment:
Try to use more div tags. It's all about boxes inside boxes inside boxes.
An example could be:

```
<div class = "millions-songs">
   <img id= "songs" src= "./images/music-icon.png">
   <h3>Millions of Songs</h3>
   <p>There are millions of songs on Spotify</p>
</div>
```